ui/playwright.config.ts

```diff
@@ -0,0 +1,38 @@
+import type { PlaywrightTestConfig } from "@playwright/test";
+import { devices } from "@playwright/test";
+
+const config: PlaywrightTestConfig = {
+  testDir: "./tests",
+  /* Maximum time one test can run for. */
+  timeout: 30_000,
```

Comment:
We could also configure the `snapshotPathTemplate` option

Reply:
yes, agreed!